sdk/healthinsights/health-insights-cancerprofiling-rest/README.md

```diff
@@ -0,0 +1,269 @@
+const initialResponse = await client.path("/oncophenotype/jobs").post(parameters);
+if (isUnexpected(initialResponse)) {
+    throw initialResponse;
+}
+
+const poller = await getLongRunningPoller(client, initialResponse);
+const cancerProfilingResult = await poller.pollUntilDone();
+if (isUnexpected(cancerProfilingResult)) {
+    throw cancerProfilingResult;
+}
+const resultBody = cancerProfilingResult.body;
+
+if (cancerProfilingResult.status === "succeeded") {
+    const results = cancerProfilingResult.results;
+    const patients = results.patients;
+    for (const patientResult of patients) {
+        console.log(`Inferences of Patient ${patientResult.id}`);
+        for (const inferences of patientResult.inferences) {
+            console.log(`Clinical Type: ${String(inferences.type)} Value: ${inferences.value}, ConfidenceScore: ${inferences.confidenceScore}`);
+            if (inferences.evidence != undefined) {
+                for (const evidence of inferences.evidence) {
+                    if (evidence.patientDataEvidence != undefined)
+                    {
+                        let dataEvidence = evidence.patientDataEvidence;
+                        console.log(`Evidence: ${dataEvidence.id} ${dataEvidence.offset} ${dataEvidence.length} ${dataEvidence.text}`);
+                    }
+                }
+            }
+        }
+    }
+} else {
+    const errors = cancerProfilingResult.errors;
+    if (errors) {
+        for (const error of errors) {
+          console.log(error.code, ":", error.message);
+        }
+    }
+}
+```
+
+## Troubleshooting
+
+### Logging
+
+Enabling logging may help uncover useful information about failures. In order to see a log of HTTP requests and responses, set the `AZURE_LOG_LEVEL` environment variable to `info`. Alternatively, logging can be enabled at runtime by calling `setLogLevel` in the `@azure/logger`:
+
+```javascript
+const { setLogLevel } = require("@azure/logger");
+
+setLogLevel("info");
+```
+
+For more detailed instructions on how to enable logs, you can look at the [@azure/logger package docs](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/core/logger).
+
+## Next steps
+<!--
+This code sample show common scenario operation with the Azure Health Insights Cancer Profiling library. More samples can be found under the [samples](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/healthinsights/azure-healthinsights-cancerprofiling/samples/) directory.
+- Infer Cancer Profiling: [sample_infer_cancer_profiling.ts](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/healthinsights/azure-healthinsights-cancerprofiling/samples/sample_infer_cancer_profiling.ts)
+-->
+
+### Additional documentation
+<!--
+For more extensive documentation on Azure Health Insights Cancer Profiling, see the [Cancer Profiling documentation](https://review.learn.microsoft.com/en-us/azure/cognitive-services/health-decision-support/oncophenotype/overview?branch=main) on docs.microsoft.com.
+-->
```

Comment:
Is this link not available ? Also, maybe need to remote the en-us in the links, so that it's convenient for people who's not using english in their browser.

Reply:
Yes I fixed the link in two README files